### PR TITLE
add debian/ubuntu packaging

### PR DESCRIPTION
**Pull request by javier-lopez**
Maybe this could be used as base to create a daily ppa for connman-ncurses

### CODE REVIEW

File: packaging/debian/rules [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+export DH_VERBOSE=1
+%:
+	dh $@
+
+override_dh_auto_configure:
+	autoreconf -i -f
+	./configure --disable-optimization --enable-debug
+
+override_dh_auto_make:
+	make -B
```
Comment (alan-mushi) [inline packaging/debian/rules]: No need for `-B` here.

File: packaging/debian/control [hunk 2]
```diff
@@ -0,0 +1,22 @@
+Source: connman-curses-git
+Maintainer: Javier López <m@javier.io>
+Section: net
+Priority: optional
+Build-Depends: debhelper  (>=9), libjson-c2, libjson-c-dev, autoconf (>=2.69), automake, libncurses5, libncurses5-dev, libdbus-1-dev, dh-exec (>= 0.12)
```
Comment (alan-mushi) [inline packaging/debian/control:5]: This project uses libjson0 (although I'm not aware of the difference between those two packages, they seem pretty close).

File: packaging/debian/connman-curses-git.install [hunk 3]
```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/dh-exec
+debug.sh => /usr/bin/connman-curses-debug.sh
```
Comment (alan-mushi) [inline packaging/debian/connman-curses-git.install]: I'm not sure why you ship this 2 lines script, it's quite basic...

File: packaging/debian/rules (hunk 1 above)
Comment (javier-lopez) [inline packaging/debian/rules]: Ok, I'm gonna remove it

File: packaging/debian/control (hunk 2 above)
Comment (javier-lopez) [inline packaging/debian/control:5]: libjson was renamed to libjson-c (https://github.com/json-c/json-c/issues/5) in debian/ubuntu libjson0 is a transitional package (who may disappear in future releases), libjson-c2 is the new package name assigned to libjson-c.

File: packaging/debian/connman-curses-git.install (hunk 3 above)
Comment (javier-lopez) [inline packaging/debian/connman-curses-git.install]: I though it could be useful to help debugging, but I'm removing it

**Comment by javier-lopez** [comment]
I just updated the push request

**Comment by jobol** [comment]
A great THANKS!

I've not tested and I trust on you. Don't be affraid to push update if needed... ^+^

**Comment by javier-lopez** [comment]
Thanks for trusting =), I've created a daily ppa (packages will be generated automatically after every commit) for connman-json-client-git, connman stable and connman-git.

https://launchpad.net/~chilicuil/+archive/ubuntu/connman-json-client-daily

**Comment by javier-lopez** [comment]
Hello,

I've been doing some more testing on the package and I found a kind of weird issue, whenever the package is created with `make -B` the program can be called on this way (which may be useful for launchers [.desktop files]).

```
exec x-terminal-emulator -title connman-curses -e /bin/bash -c 'eval /usr/bin/connman-curses'
```

In the other hand if the package is created with simple `make` (just at it's now) the above command fails.

**Pull request merged**